website/docs/guides/markdown-features/markdown-features-math-equations.mdx

```diff
@@ -49,92 +49,98 @@ $$
 
 </BrowserWindow>
 
-## Configuration {#configuration}
-
-To enable KaTeX, you need to install `remark-math` and `rehype-katex` plugins.
-
-```bash npm2yarn
-npm install --save remark-math@6 rehype-katex@7
-```
-
-:::warning
-
-Make sure to use `remark-math 6` and `rehype-katex 7` for Docusaurus v3 (using MDX v3). We can't guarantee other versions will work.
-
-:::
-
-Those 2 plugins are now [**only available as ES Modules**](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c). To simplify usage, it is recommended to use an [**ES Modules**](https://flaviocopes.com/es-modules/) config file:
-
-```js title="ES module docusaurus.config.js"
-// highlight-start
-import remarkMath from 'remark-math';
-import rehypeKatex from 'rehype-katex';
-// highlight-end
-
-// highlight-start
-export default {
-  presets: [
-    [
-      '@docusaurus/preset-classic',
-      {
-        docs: {
-          path: 'docs',
-          // highlight-start
-          remarkPlugins: [remarkMath],
-          rehypePlugins: [rehypeKatex],
-          // highlight-end
-        },
-      },
-    ],
-  ],
-};
-```
+## Enabling math equations {#configuration}
+
+Enable KaTeX:
+
+1. Install the `remark-math` and `rehype-katex` plugins:
+
+   ```bash npm2yarn
+   npm install --save remark-math@6 rehype-katex@7
+   ```
+   
+   :::warning
+   
+   Make sure to use `remark-math 6` and `rehype-katex 7` for Docusaurus v3 (using MDX v3). We can't guarantee other versions will work.
+   
+   :::
+
+1. These 2 plugins are [**only available as ES Modules**](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c). We recommended to use an [**ES Modules**](https://flaviocopes.com/es-modules/) config file:
```

Comment:
Is this numbering enforced by any linting we have? If Prettier leaves them alone, I would suggest using `2.`. We don't use `1.` everywhere because that looks awkward :)

Reply:
Fair enough, but I would highly consider having steps in task-style documentation sections. This is standard practice (see [github](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/checking-for-existing-ssh-keys#checking-for-existing-ssh-keys), [microsoft](https://learn.microsoft.com/en-us/azure/ai-services/multi-service-resource?tabs=macos&pivots=azportal#create-a-new-multi-service-resource), and anyone else), and considering how heavy the docusuaurus docs are on contributions, I would expect ppl to potentially add steps in the middle of certain tasks to provide finer guidance from time to time, and if you have deal with numbering by hand, that's not fun. 

Especially in the docs as code approach, it's so easy to build the output that the source shouldn't matter for readability all that much, but convenience should rule. 

Happy to update the numbering if you think it's the best way to go.

Reply:
Yes, please update the numbers. Even if someone adds a number in between, it's obvious enough if the numbers don't match, and we would just update all following numbers.